alibi_detect/saving/schemas.py

```diff
@@ -74,9 +73,9 @@ class Config:
 
 
 class MetaData(CustomBaseModel):
-    version: str = __version__
-    config_spec: str = __config_spec__
-    version_warning: Optional[bool] = None  # None instead of False as same effect in logicals but hides from config
+    version: str
+    config_spec: str
+    version_warning: bool = False
```

Comment:
`version_warning` changed because setting to `None` no longer hides from config (since we now convert `None` to `"None"` when writing to `config.toml`, instead of it being removed).